test/test_makefiles.py

```diff
@@ -0,0 +1,30 @@
+"""
```

Comment:
Maybe rename this to `test_makefiles_present.py`, to reduce confusion with `test_makefile_dependencies`?